workshops/views.py

```diff
@@ -496,6 +509,13 @@ def membership_create(request, org_domain):
     return render(request, 'workshops/generic_form.html', context)
 
 
+class MembershipCreate(OnlyForAdminsMixin, PermissionRequiredMixin,
+                 CreateViewContext):
+    permission_required = 'workshops.add_membership'
+    model = Membership
+    form_class = MembershipForm
```

Comment:
@pbanaszkiewicz We already have a view called `membership_create` [here](https://github.com/swcarpentry/amy/blob/develop/workshops/views.py#L464).

We should extend it to allow populating the `organization` field based on a URL or GET parameter.